<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CParticleLinkDialog</class>
 <widget class="QDockWidget" name="CParticleLinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Link to skeleton</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTreeView" name="treeView">
      <property name="toolTip">
       <string>Choose bone to stick particle system</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="linkPushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Link</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="unlinkPushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Unlink</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
